api/v1beta1/kustomization_types.go

```diff
@@ -48,6 +48,11 @@ type KustomizationSpec struct {
 	// +required
 	Interval metav1.Duration `json:"interval"`
 
+	// The interval at which to retry a previously failed reconciliation.
+	// When not specified, it defaults to the Interval value.
```

Comment:
```suggestion
	// Defaults to the Interval value.
```

(easier to comprehend)